src/api-v1.js

```diff
@@ -103,10 +106,17 @@ api.declare({
             url: url,
           });
         }
+        // TODO: Should only status === 'error' be considered a miss?  A file
+        // that's never been requested before could never be in the cache, so
+        // it not being there really is not an error case
+        this.monitor.count(`${service}.${region}.cache-miss`, 1);
+      } else {
+        this.monitor.count(`${service}.${region}.cache-hit`, 1);
       }
 
       if (result.status === 'present') {
         debug(`${logthingy} is present`);
+        this.monitor.measure(`${service}.${region}.cache-hit.time`, new Date() - startTime);
```

Comment:
wouldn't `this.monitor.count(`${service}.${region}.cache-hit`, 1);` be better located here?

or is `cache-hit` also counting status == errors?

Regardless, I would rename this to `cache-hit-duration` or `cache-hit-time`